### DISCUSSION TITLE
Sobre task 01, Lab 6.02

### DISCUSSION THREAD

**Discussion by maguiargil** [rramosp/ai4eng.v1#258] [06 - MÉTODOS NO SUPERVISADOS]
Cordial saludo. Espero que se encuentren muy bien,

Con respeto, quisiera llamar a atención de lo siguiente:

![image](https://github.com/rramosp/ai4eng.v1/assets/55927354/a01425a3-306a-47da-93f8-7f8f20288822)

¿Es obligatoria la coincidencia exacta en cantidad de decimales para validar el resultado?

**Comment by Felipe-RA** [comment]
Saludos @maguiargil 

Recuerda que debes entregar un arreglo de numpy 1D. 

Adicionalmente, recuerda que puedes redondear los arreglos flotantes con `np.round()` para que coincidan con el número de decimales mostrados en los ejemplos.

Si tienes  más preguntas estoy pendiente, me comentas si con esto solucionas tu problema.

¡Hasta pronto!
